### PR TITLE
PT-959 Enable SSSD service autorestart

### PR DESCRIPTION
**Pull request by antonu17**

### CODE REVIEW

**Comment by brunoliveira8** [comment]
👍

**Comment by julyate** [comment]
+1

**Comment by vgarvardt** [comment]
👍

File: run_tests.sh [hunk 1]
```diff
@@ -61,7 +61,7 @@ then
 fi
 virtualenv venv
 . ./venv/bin/activate
-pip install ansible docker-py molecule
+pip install -r requiremets.txt
```
Comment (michael-robertson) [inline run_tests.sh]: requiremets? :)

Comment (antonu17) [reply run_tests.sh]: Naturlich

**Comment by bghellofresh** [comment]
+1

**Comment by aleksandrzhiliaev** [comment]
+1

**Comment by vgarvardt** [comment]
👍

**Pull request merged**